sources/packages/backend/apps/api/src/services/education-program-offering/education-program-offering.service.ts

```diff
@@ -996,13 +1003,16 @@ export class EducationProgramOfferingService extends RecordDataModelService<Educ
     const currentOffering = await this.getOfferingToRequestChange(
       offeringId,
       OfferingStatus.Approved,
-      programId,
-      locationId,
+      {
+        programId: programId,
+        locationId: locationId,
+        isIncludeInActiveProgram: false,
+      },
```

Comment:
Same here. Validation doesn't need to cascade to next layer unless there is a need.